### ISSUE TITLE
Documentation ? 

### ISSUE THREAD

**Issue by fxfactorial** [capitalone/react-native-pathjs-charts#146]
Is there any documentation ? At least to give the schema of the objects for config of the charts? 

Example, I want the x axis labels to be just horizontal, the example makes labels that are like 120 degrees.

**Comment by superandrew213** [comment]
See the [wiki](https://github.com/capitalone/react-native-pathjs-charts/wiki ) and example app. It's pretty straight forward.

**Comment by fxfactorial** [comment]
@superandrew213 Thanks for quick reply. Yea I see the examples but this examples have code rot eventually over time. It would be much better if you documented exactly what kinds of keys and values are appropriate. 

I don't see what I need to do to change in configs to make the labels be horizontal. Can you help me with that?

**Comment by superandrew213** [comment]
I'm assuming you are referring to the label rotation for the BarChart?

There is no setting for the label rotation yet. I needed this too so I added a rotate option. See my PR https://github.com/capitalone/react-native-pathjs-charts/pull/144. You can use that until it gets merged.

Add it like this:
```
    axisX: {
      showAxis: true,
      showLines: true,
      showLabels: true,
      showTicks: true,
      zeroAxis: false,
      orient: 'bottom',
      label: {
        fontFamily: 'Arial',
        fontSize: 8,
        fontWeight: true,
        fill: '#34495E',
        rotate: 0,  // <==== here
      }
    },
```

**Comment by fxfactorial** [comment]
Yes, nice.

**Comment by marzolfb** [comment]
Agreed the documentation is lacking and describing all the configuration options in more detail on the wiki is desirable. As is a typical response, its just a matter of finding time to make it happen. It's a shame Github doesn't have a natural PR mechanism to allow for community-driven content for wikis like they do normally for the repository. 

I will leave this open as a reminder to get it done and close it once there is some base level of documentation describing configuration options as you suggest.

**Comment by Jacse** [comment]
Many repositories use `README` for documentation purposes. Changing the documentation from wiki to readme would enable improvements from the community and speed things up significantly. While the wiki naturally is a better place for documentation it seems to be a reasonable trade-off. What do you think @marzolfb?

**Comment by katscott** [comment]
@marzolfb and I have been talking offline about this for a while.  The wiki was nice in that it let the documentation be searched separately.  Unfortunately you lose the ability to have pull requests (very common complaint of the wiki around github). 
The readme also would get very bloated if all the documentation was there (even as small as it is right now). We had been discussing putting them into a docs folder, even though we would lose the search ability the wiki gives us.
Regardless, there isn't any good solution here, but for now we've moved it into a docs folder that has been configured as the gh-pages location.  This will let it be edited in a PR at least.

**Comment by Jacse** [comment]
@katscott that seems like a reasonable solution.

**Comment by smitthakkar1** [comment]
This is really helpful guys. Thanks
I want to make bars horizontal. I am not able to do it. Can anyone help ?
Thanks
